docs/dev_mode.md

```diff
@@ -46,18 +32,10 @@ Once the agents are running, you can make changes to the agent's packages as wel
 The trigger is caused by any python file closing in either `open-autonomy/packages` or `open-aea/` directory. So even if you haven't made any change and still want to restart the agent, just open any python file press `ctrl+s` or save it from file menu and it will trigger the restart.
 
 
-## Quick start for oracle
-
-We have a single make target for the oracle app, just run the following command and it will build and run the oracle.
-```bash
-make run-oracle-dev
```

Comment:
Somewhat unrelated to the PR, but I spotted it here. Please remove the make commands in these files too:
docs/application_deployment.md
docs/autonomy.md
docs/benchmarks.md
docs/counter_example.md
docs/hot_reload.md
docs/using_stack_deployment.md

In cases where the commands are needed, a sample makefile with the commands should be added to the docs. Basically, it must be possible to use it without pulling the repo.